app/views/steps/dwp/confirm_result/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+  <%= govuk_error_summary(@form_object) %>
+
+<div class="govuk-grid-row app-inverted--card app-inverted-text">
+  <div class="govuk-grid-column app-inverted-text">
+    <h1 class="govuk-heading-l app-inverted-text"><%= t '.heading' %></h1>
+  </div>
+</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_radio_buttons_fieldset(:confirm_result, legend: { tag: 'h1', size: 'l' }) do %>
```

Comment:
Let's use `govuk_collection_radio_buttons` (like in the other view `dwp/confirm_details/edit.html.erb`) so we don't have to do a loop on the choices.

Additionally this page has two `h1`. If you use the same code as in the other view this will also be fixed I believe.